### ISSUE TITLE
Manually labeling - how to input?

### ISSUE THREAD

**Issue by koenie64** [zhengthomastang/2018AICity_TeamUW#11]
Hi!
We are trying to reproduce /reuse the scripts. However we get stuck at the manual labelling. In the darknet aicity.data, a reference is made to:
train  = /home/ipl_gpu/Aotian/darknet2/data/aicity/train.txt
valid  = /home/ipl_gpu/Aotian/darknet2/data/aicity/validation.txt

are these the class (sedan etc.) labels? How should we format those files?
Thjanks

**Comment by zhengthomastang** [comment]
Hi @koenie64 . Thank you for using our software package. As the datasets of the 2018 AI City Challenge belong to NVIDIA and they are not available currently, you cannot train the model. But our pretrained model is provided for testing. Note that it may not perform well on other datasets. Feel free to use other state-of-the-art detectors like Faster R-CNN, SSD, YOLOv3, etc.

**Comment by koenie64** [comment]
Hi!
We understand that - we will use the new 2019 dataset. I was more
interested in the structure of the manual labeling (format of the text
file) so that we can reuse the script with new data.

Thanks! Koen

**Comment by zhengthomastang** [comment]
Hi @koenie64! For the 2019 AI City Challenge, I do not suggest you to train a detector using the provided ground truth of MTMC tracking, because only vehicles passing through multiple cameras are annotated. In other words, a lot of cars, especially those parking by the streets, are not labeled. You will not get good performance if training on those data. However, the performance of the provided detection baselines should be good enough, especially SSD512. Let me know if you still have any questions.

**Comment by Ujang24** [comment]
> Hi @koenie64 . Thank you for using our software package. As the datasets of the 2018 AI City Challenge belong to NVIDIA and they are not available currently, you cannot train the model. But our pretrained model is provided for testing. Note that it may not perform well on other datasets. Feel free to use other state-of-the-art detectors like Faster R-CNN, SSD, YOLOv3, etc.


Could you please mention the steps for using YOLOv3 for the object detector?

**Comment by zhengthomastang** [comment]
@Ujang24 The easiest way is to run the pretrained models (ImageNet or COCO) and change the output format to match with our definition. Or you can follow the tutorial of YOLOv3 to train your own models.

**Comment by Ujang24** [comment]
> @Ujang24 The easiest way is to run the pretrained models (ImageNet or COCO) and change the output format to match with our definition. Or you can follow the tutorial of YOLOv3 to train your own models.

Thanks. It helps. Sorry for the late reply.
